test/e2e/far_e2e_test.go

```diff
@@ -116,30 +117,41 @@ var _ = Describe("FAR E2e", func() {
 			nodeBootTimeBefore, err = e2eUtils.GetBootTime(clientSet, nodeName, testNsName, log)
 			Expect(err).ToNot(HaveOccurred(), "failed to get boot time of the node")
 
-			// create tested pod, and save its creation time
-			// it will be deleted by FAR CR
+			// create tested pod which will be deleted by the far CR
 			pod = e2eUtils.GetPod(nodeName, testContainerName)
 			pod.Name = testPodName
 			pod.Namespace = testNsName
 			Expect(k8sClient.Create(context.Background(), pod)).To(Succeed())
-			log.Info("Tested pod has been created", "pod", testPodName)
-			creationTimePod = metav1.Now().Time
 			DeferCleanup(cleanupTestedResources, pod)
 
 			// set the node as "unhealthy" by disabling kubelet
 			makeNodeUnready(selectedNode)
 
+			startTime = time.Now()
```

Comment:
I wonder if this is accurate enough. It is taking into account the time to create the object (which is variable) other than the actual initial instant of the remediation.

Reply:
> It is taking into account the time to create the object (which is variable) other than the actual initial instant of the remediation.

Are you suggesting just "storing" the time after creating far CR (after line 131) instead of before the creation? I believe it is negligible, but I don't mind doing so

Reply:
I was thinking of getting the cr creation time, but you're right that the difference should be minimal